helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -12,6 +12,12 @@ rules:
   - customresourcedefinitions
   verbs:
   - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions/status
+  verbs:
+  - "*"
```

Comment:
Without permission to update the CRD status the operator crashes if there is already an app CRD.